charts/connectivity-monitor/templates/ds.yaml

```
@@ -0,0 +1,68 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: connectivity-exporter-config
+  namespace: {{ .Values.namespace }}
```

```suggestion
  namespace: {{ .Release.namespace }}
```
is maybe simpler because the namespace does not have to be configured as a "value".
https://helm.sh/docs/chart_template_guide/builtin_objects/